### PR TITLE
Stars

### PR DESCRIPTION
**Pull request by claire123c**
- Added star rating functionality and star image
- Added tests for star rating
- Fixed edge case bugs with image gallery (onClickThumbnail shouldn't allow arrows to render if only one image in array)

### CODE REVIEW

File: client/src/components/overview/Overview.jsx [hunk 1]
```diff
@@ -19,7 +28,17 @@ function Overview() {
   const getStyles = () => {
     axios.get(`/products/${productNum}/styles`)
       .then((response) => {
-        setProductData(response.data.results);
+        setStyleData(response.data.results);
+        console.log(response.data.results);
```
Comment (johnamielyasis) [inline client/src/components/overview/Overview.jsx:32]: is this console.log still needed?

**Review by johnamielyasis — APPROVED** [review]
lgtm
